openpype/modules/deadline/plugins/publish/submit_houdini_render_deadline.py

```diff
@@ -88,7 +88,7 @@ def get_job_info(self):
             "AVALON_APP_NAME",
             "OPENPYPE_DEV",
             "OPENPYPE_LOG_NO_COLORS",
-            "OPENPYPE_VERSION"
```

Comment:
@antirotor is this ok to be removed. It seemed like some old code to me.

Reply:
Yup, should be removed cause it'll be added if running from a build.

Reply:
This should be removed as its a separate PR now.